src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/AddArtworksModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react"
+import { Flex, Text, ModalDialog, Spacer, Button } from "@artsy/palette"
+import { SortFilter } from "Components/SortFilter"
+import { ArtworksList, TEMP_ARTWORKS } from "./ArtworksList"
+import { useTranslation } from "react-i18next"
+
+interface AddArtworksModalProps {
+  visible: boolean
+  onClose: () => void
+  onComplete: () => void
+  listName: string
+}
+
+const AddArtworksModal: React.FC<AddArtworksModalProps> = ({
+  listName,
+  onClose,
+}) => {
+  const { t } = useTranslation()
+
+  const handleSave = () => {
+    onClose()
+  }
+
+  return (
+    <ModalDialog
+      m={["0", "auto"]}
+      dialogProps={{
+        width: ["100%", 700],
```

Comment:
it's better to replace `width` on `minWidth`

Reply:
hey @dimatretyak with `minWidth` my modal takes the whole width of the screen, so I left it like that